app/routes.js

```diff
@@ -1,7 +1,18 @@
-// const controller = require('./controllers/controller');
+const indexController = require('./controllers/index'),
+  userController = require('./controllers/users'),
+  generics = require('./middlewares/generics');
 
 exports.init = app => {
-  // app.get('/endpoint/get/path', [], controller.methodGET);
-  // app.put('/endpoint/put/path', [], controller.methodPUT);
-  // app.post('/endpoint/post/path', [], controller.methodPOST);
+  app.use(generics.logRequestInformation);
```

Comment:
this file is only for routes, and middlewares that apply to specific routes, if you want to use a middlewares for all of them with `use` put it in `app.js`. In any case, there is no requirement to log every request, just log errors and such

Reply:
Can I maintain it? To get more information about requests. I will move it to app.js.